docs/annexes/cross-reference.md

```diff
@@ -0,0 +1,246 @@
+# Annex I: Cross referencing in SPDX 3 (Informative)
+
+This document will walk though how to refer to SPDX Elements across documents
+(e.g. cross reference).
+
+If you do would like to construct the complete example documents from this
+Markdown file, use the following command:
+
+```shell
+cat cross-reference.md | awk 'BEGIN{flag=0} /^```json/, $0=="```" { if (/^---$/){flag++} else if ($0 !~ /^```.*/ ) print $0 > "doc-" flag ".spdx.json"}'
+```
+
+## Linking via spdxId
+
+It is frequently desired (and necessary) to reference an SPDX 3
+[Element][Class_Element] that lives in one document from another. Since SPDX
+documents are valid [JSON-LD][JSON_LD] documents, linking elements together can
+be as simple as referencing the spdxId of one element from another (in the same
+way that doing so within a document links Elements together. For example,
+assume we have this document that contains a [Person][Class_Person] we want to
+reference in another document:
+
+```json
+{
+    "@context": "https://spdx.org/rdf/3.0.0/spdx-context.jsonld",
+    "@graph": [
+        {
+            "type": "Person",
+            "spdxId": "http://spdx.example.com/Person/JoshuaWatt",
+            "creationInfo": "_:creationinfo",
+            "name": "Joshua Watt",
+            "externalIdentifier": [
+                {
+                    "type": "ExternalIdentifier",
+                    "externalIdentifierType": "email",
+                    "identifier": "JPEWhacker@gmail.com"
+                }
+            ]
+        },
+        {
+            "type": "CreationInfo",
+            "@id": "_:creationinfo",
+            "specVersion": "3.0.0",
+            "createdBy": [
+                "http://spdx.example.com/Person/JoshuaWatt"
+            ],
+            "created": "2024-03-06T00:00:00Z"
+        },
+        {
+            "type": "SpdxDocument",
+            "spdxId": "http://spdx.example.com/Document1",
```

Comment:
I think we should use more realistic and potentially globally unique IDs.

Reply:
What classifies as "more realistic"?

Reply:
something that could be globally unique. Maybe what I am missing is guidance on how to create IDs. I just know hand crafted non-realistic examples. Some using `urn:..`. I think this might be something for a different document but I it is a gap.

Reply:
Ya, I don't mind making it better, I'm just not clear what "better" would be in this case. There's not currently great guidance on this other than that it should be "unique"